examples/cli.rs

```diff
@@ -476,23 +499,25 @@ async fn run<C: Store + MessageStore>(subcommand: Cmd, config_store: C) -> anyho
             }
         }
         Cmd::GetGroup { group_master_key } => {
+            let( master_key, key ) = group_master_key;
             let manager = Manager::load_registered(config_store)?;
-            let group = manager.get_group_v2(group_master_key).await?;
+            let group = manager.get_group_v2(master_key).await?;
```

Comment:
I'm not super sure why you'd have to do it like this? Is it because you want to use the master key as a string key in the storage backend?

Reply:
The `proto::group` has no field for the master_key, so how should we store the masterkey? Either by adding a new field or using it as key in the tree.

Reply:
I am unsure, it was just a shot in the dark. First i looked into the `groups_v2::Group` but this doesn't serialize into json or a protobuf, so I tried it this way.